builder/googlecompute/builder_acc_test.go

```diff
@@ -284,3 +286,101 @@ func TestAccBuilder_WithMultipleDisks(t *testing.T) {
 	}
 	acctest.TestPlugin(t, testCase)
 }
+
+func TestAccBuilder_ImageArchVariations(t *testing.T) {
```

Comment:
As I understand this this adds four new executions of builds to GCP, I am not sure of the existing status of parallelizing these tests but it may be worth investigating when adding this many tests, also it looks like acceptance tests are currently failing https://github.com/hashicorp/packer-plugin-googlecompute/actions/runs/10275873966/job/28435328892

Reply:
Addressed in a subsequent PR